src/main/kotlin/org/opensearch/replication/task/index/IndexReplicationTask.kt

```diff
@@ -627,13 +624,15 @@ open class IndexReplicationTask(id: Long, type: String, action: String, descript
         return MonitoringState
     }
 
-    private fun findAllReplicationFailedShardTasks(clusterState: ClusterState)
+    private fun findAllReplicationFailedShardTasks(followerIndexName: String, clusterState: ClusterState)
```

Comment:
nit : we already have `followerIndexName` as a member .

Reply:
Keeping as is for now